### PR TITLE
Commissary remap

### PR DESCRIPTION
**Pull request by Don2137**
## About The Pull Request
I swapped the Commissary with the Auxiliary Infirmary and made it somewhat bigger. Auxinf also gets a bootleg chem setup. That's about it.

Previous PR was completely wrong on top of having map issues, now everything should be fine
## Why It's Good For The Game
RP areas are always good.
## Did You Test It?
Yes
## Authorship
me
## Changelog

:cl:
maptweak: Remapped the Commissary and the Auxiliary Infirmary
/:cl:

### CODE REVIEW

**Comment by Anton-Kr** [comment]
Large amount of changed lines is indicative of having not run map merger. Contact qqqq on discord if assistance is needed with the process.

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Run map merger

**Comment by Anton-Kr** [comment]
I think the original intent was for traders to use commissary which is why they are next to each other, but ehh I haven't seen that be used much. If checks are good, this should be approved.

**Comment by Don2137** [comment]
Unsure why this is failing

**Comment by Anton-Kr** [comment]
```
[02:00:00] Runtime in code/game/machinery/embedded_controller/docking_program.dm,69: Docking controller tag admin_shuttle_dock_airlock had multiple associated programs.
  proc name: crash at (/proc/crash_at)
  src: null
  call stack:
  crash at("Docking controller tag admin_s...", "code/game/machinery/embedded_c...", 69)
  /datum/computer/file/embedded_... (/datum/computer/file/embedded_program/docking/airlock): New(the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port))
  /datum/computer/file/embedded_... (/datum/computer/file/embedded_program/docking/airlock): New(the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port))
  the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port): Initialize(1)
  the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port): Initialize(1)
  the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the docking port controller (/obj/machinery/embedded_controller/radio/airlock/docking_port), /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms()
  Atoms (/datum/controller/subsystem/atoms): Initialize(795478)
  Atoms (/datum/controller/subsystem/atoms): DoInitialize(795478)
  Master (/datum/controller/master): Initialize(10, 0)
```

Not sure if you touched airlocks here but that's the likely issue. If you cannot find it bug me tomorrow and I'll have a look personally.

**Comment by Don2137** [comment]
Didn't touch that docking port, no idea what's wrong. Map looks fine too.

**Comment by Anton-Kr** [comment]
I'll take a look tomorrow or in two days to help you resolve the issue. Please feel free to ping me if (more likely when) I forget.

**Comment by Rafflesia7793** [comment]
If the commissary was touched there's a decent chance the docking controller next to it got clipped: 
image

**Comment by Don2137** [comment]
> If the commissary was touched there's a decent chance the docking controller next to it got clipped: image

This one is fine, it's the aft starboard one that's runtiming.

**Comment by Rafflesia7793** [comment]
Might be related but the Docking Controller inside the airlock next to the commissary is configured for the airlock that's now throwing errors:
image
Admin Shuttle is the Aft Starboard airlock. This is the fore starboard which is usually labelled merchant shuttle.

File: maps/torch/torch2_deck4.dmm [hunk 1]
```diff
@@ -1421,13 +1481,21 @@
 	dir = 8;
 	pixel_x = 21
 	},
+/obj/machinery/embedded_controller/radio/simple_docking_controller{
+	dir = 4;
+	id_tag = "admin_shuttle_dock_airlock";
+	pixel_x = -24
+	},
```
Comment (Rafflesia7793) [inline maps/torch/torch2_deck4.dmm]: This is what's causing your testing failure currently.

**Pull request closed without merging**